### DISCUSSION TITLE
Quick Start Mode fails due to isbot

### DISCUSSION THREAD

**Discussion by adamcstephens** [remix-run/remix#8411] [Q&A] [answered]
Hi, following the guide at https://remix.run/docs/en/2.4.1/start/quickstart

I am unable to start remix-serve

```
─❯ npx remix-serve build/index.js
file:///Users/adam/projects/my-remix-app/build/index.js?t=1704220131429.254:15
import isbot from "isbot";
       ^^^^^
SyntaxError: The requested module 'isbot' does not provide an export named 'default'
```

Am I doing something wrong, or is this a recently introduced bug?

**Comment by alexanderson1993** [comment]
See the notice here: https://www.npmjs.com/package/isbot#migrate-to-version-4-today

Change your import to this:

```js
import { isbot } from "isbot";
```

**Reply by adamcstephens** [reply]
That allows me to start remix-serve, but this import is in `build/index.js` so it's reverted if I re-run `remix build`.

**Reply by adamcstephens** [reply]
It looks like this may be the fix we need: https://github.com/remix-run/remix/pull/8385

**Reply by kiliman** [reply]
Another option is to use `npx remix reveal`. This will create the _entry.server.tsx_ file that you can then edit to change the import.

**Reply by lb-** [reply]
It looks like https://github.com/remix-run/remix/pull/8385 has been reverted.

Ran into this issue today, first time trying to get Remix started locally.

Maybe the docs could be updated to use supported versions for each docs release?

For example.

```
npm i @remix-run/node @remix-run/react @remix-run/serve isbot@3.8.0 react react-dom
```

**Reply by fzsf163** [reply]
this is the solution. thank you 💖💖💖

**Comment by kasperkronborg** [comment, marked as answer]
I believe this could also be solved temporarily by running `npm install isbot@3.8.0`

**Reply by lb-** [reply]
This helped me get this working, thank you.

**Comment by philfuster** [comment]
Followed tutorial from [here](https://remix.run/docs/en/main/start/tutorial#setup) and ran into this issue today. :(
